### ISSUE TITLE
Retain protocols that inherits JSExport protocol

### ISSUE THREAD

**Issue by sdc-78** [peripheryapp/periphery#414]
The [`JSExport`](https://developer.apple.com/documentation/javascriptcore/jsexport) protocol is provided by [`JavaScriptCore`](https://developer.apple.com/documentation/javascriptcore) to expose Objective-C and Swift code to JavaScript code. The code (properties, methods) declared in a protocol that inherits `JSExport` is intended to be called by JavaScript code, which Periphery cannot see. So such a protocol should be retained by Periphery.

**Comment by ileitch** [comment]
Using `--retain-objc-accessible` should correctly retain these protocols, though it might actually be nice to retain JS accessible without having to use `--retain-objc-accessible`. An option like `--retain-js-accessible` might be appropriate.

**Comment by sdc-78** [comment]
You're right. I hadn't thought of using --retain-objc-accessible option. It does the job. A --retain-js-accessible option would be nice to have.
